src/components/login-view/login-view.jsx

```diff
@@ -0,0 +1,78 @@
+<<<<<<< Updated upstream
+=======
```

Comment:
i guess u have a merge conflict here. 
this should be probably removed.

```
<<<<<<< Updated upstream
=======
```

Reply:
hmm u have 2 versions currently in ur file, the updated stream version and the stashed changes. which u wanna keep? u can jus delete the ones u dont want to keep.